components/hab/src/command/supportbundle.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2016-2018 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use chrono::Local;
+use common::ui::{Status, UIWriter, UI};
+use error::{Error, Result};
+use flate2::Compression;
+use flate2::write::GzEncoder;
+use hcore::fs::FS_ROOT_PATH;
+use hcore::os::net::hostname;
+use std::env;
+use std::error::Error as StdErr;
+use std::fs;
+use std::fs::File;
+use std::path::{Path, MAIN_SEPARATOR};
+use std::process;
+use tar;
+
+pub fn lookup_hostname() -> Result<String> {
```

Comment:
Minor, but since this is only used once, and only in this module, it could be private, or folded into the callsite directly.